### ISSUE TITLE
Screenreader: Menu for elevating users previleges not beeing accessible

### ISSUE THREAD

**Issue by WestphalDenn** [bigbluebutton/bigbluebutton#12574]
Description: 
The Screenreader does not see the Menu when I try to give individual Users previleges. That only effects Firefox. In Chrome that works fine. And it only hapens with this menu. Other menus work as expected. 

Current behaviour:
Using a Screenreader I click on the Button which opens the Menu. In this menu I could mute participants or give access to the whiteboard. However the menu can't be read by the Screenreader. It is as if the menu doesn't exists for the Screenreader. I can acess it when I perform OCR but that shouldn't be necessary. 

Expected behaviour: 
This menu should behave as the other menus in BBB. They work fine. 

Personal notes: 
I thank everyone who works on making BBB as accessible as it already is. It really is a joy to use as a blind guy.

**Comment by antobinary** [comment]
Closed in BBB 2.4 via #12745
